docs/monitoring/spm-faq.md

```diff
@@ -780,8 +802,9 @@ use.
 
 **Can SPM client use HTTP instead of HTTPS to send metrics from my servers?**
 
-Yes, although we recommend using HTTPS. Since version 1.22.0, SPM
-client by default uses HTTPS to send metrics data to Sematext. If you
+Yes, although we recommend using HTTPS. 
+
+SPM client by default uses HTTPS to send metrics data to Sematext. If you
```

Comment:
SPM client - in situations like this we could simply say "Sematext agents" to refer to all of them collectively, ha?